<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>863</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1360</width>
    <height>798</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1360</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Repetitive Sequence Analysis and Correction</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../media/dna.ico</normaloff>../media/dna.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_sequence">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Sequence</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="2">
       <widget class="QTextEdit" name="textEdit_seq">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="bttn_openSeq">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bttn_save">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bttn_clearSeq">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>581</width>
          <height>95</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>581</width>
          <height>107</height>
         </size>
        </property>
        <property name="title">
         <string>Randomization</string>
        </property>
        <widget class="QPushButton" name="bttn_randomize">
         <property name="geometry">
          <rect>
           <x>424</x>
           <y>63</y>
           <width>150</width>
           <height>28</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Randomize</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_randomization">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>29</y>
           <width>100</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Randomization %</string>
         </property>
        </widget>
        <widget class="QSlider" name="hSlider_rand">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>22</y>
           <width>371</width>
           <height>32</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>2</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_rand">
         <property name="geometry">
          <rect>
           <x>515</x>
           <y>25</y>
           <width>51</width>
           <height>24</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>50</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_analysis">
     <property name="minimumSize">
      <size>
       <width>624</width>
       <height>559</height>
      </size>
     </property>
     <property name="title">
      <string>Analysis</string>
     </property>
     <widget class="QGroupBox" name="groupBox_analysisOptions">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>26</y>
        <width>605</width>
        <height>131</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>605</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>605</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string>Analysis Options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="3">
        <widget class="QLabel" name="label_maxSubSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>32</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>14</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSlider" name="hSlider_maxSubSize">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>40</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>14</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_minText">
         <property name="text">
          <string>Minimum Substring Size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_maxText">
         <property name="text">
          <string>Maximum Substring Size</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSlider" name="hSlider_minSubSize">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>40</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_minOcc">
         <property name="text">
          <string>Minimum Occurance (%)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_minSubSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>32</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSlider" name="hSlider_minOcc">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_minOccVal">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>50</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_substring">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>386</y>
        <width>605</width>
        <height>411</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>605</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>610</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Substring Analysis</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="table_subString">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="autoScrollMargin">
          <number>16</number>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>true</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>10</number>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>90</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>185</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>SubString</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Count</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>% of Sequence</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_seqAnalysis">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>165</y>
        <width>605</width>
        <height>211</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Sequence Analysis</string>
      </property>
      <widget class="QGroupBox" name="groupBox_amino">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>90</y>
         <width>581</width>
         <height>111</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>581</width>
         <height>111</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>581</width>
         <height>111</height>
        </size>
       </property>
       <property name="title">
        <string>Animo Acid Sequence</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTextEdit" name="textEdit_aminoSeq">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_char">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>30</y>
         <width>93</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Character Count</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_gc">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>60</y>
         <width>100</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>GC Richness (%)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_charVal">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_gcVal">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="bttn_analyze">
      <property name="geometry">
       <rect>
        <x>467</x>
        <y>804</y>
        <width>150</width>
        <height>28</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Analyze Seqence</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hSlider_maxSubSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_maxSubSize</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>983</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>1201</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSlider_minSubSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_minSubSize</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>983</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>1201</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSlider_rand</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_rand</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>712</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>713</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSlider_minOcc</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_minOccVal</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1080</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>1300</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
